### ISSUE TITLE
New room list accessibility regression

### ISSUE THREAD

**Issue by t3chguy** [vector-im/riot-web#14467]
due to the move of the breadcrumbs to be under the search, now it hierarchically makes a lot less sense, if anything they should be re-ordered in CSS, or the search filter should also apply to the breadcrumbs and some tabindexes need tweaking

**Comment by turt2live** [comment]
We claimed this was fixed on an earlier call. Further work to be explored in https://github.com/vector-im/riot-web/issues/14660
